### PR TITLE
Create reflection-milestone4.md

### PR DESCRIPTION
**Pull request by nobbynguyen**

### CODE REVIEW

File: doc/reflection-milestone4.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- The dashboard can only utilize the movie part of the dataset. It could have been better if we also included `TV Show`.
+- The number of features are limited in this dataset. We could have included more interactive widgets if we had more features of the data to work on such as `vote counts`, `budget`, `revenue`, `profit`
+-  It would be great if we can implement completely reproducibility. For example, this dashboard will not get updated if Kaggle/Netflix releases more recent dataset on their website.
+
+
+## 6. Potential improvements
+
+If we had more time in the future, we could implement the following features:
+
+-   Implement complete reproducibility. This is most especially important for the underlying dataset so that the dashboard gets updated as soon as the Vancouver Police Department releases new crimes data.
```
Comment (lynnwbl) [inline doc/reflection-milestone4.md]: > Vancouver Police Department releases new crimes data.

Sorry I don't get what this line means? Could it be a typo?

File: doc/reflection-milestone4.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- The documentation was very clear and intuitive for users to use and follow.
+
+
+## 5. NetViz: Dashboard App Limitations
+
+The limitations of our dashboard include:
+
+- The dashboard can only utilize the movie part of the dataset. It could have been better if we also included `TV Show`.
+- The number of features are limited in this dataset. We could have included more interactive widgets if we had more features of the data to work on such as `vote counts`, `budget`, `revenue`, `profit`
+-  It would be great if we can implement completely reproducibility. For example, this dashboard will not get updated if Kaggle/Netflix releases more recent dataset on their website.
```
Comment (lynnwbl) [inline doc/reflection-milestone4.md]: > completely reproducibility

complete reproducibility

File: doc/reflection-milestone4.md (hunk 1 above)
Comment (nobbynguyen) [reply doc/reflection-milestone4.md]: Yeah it is. Sorry. I upload the wrong one.

**Review by lynnwbl — COMMENTED** [review]
Hey, please refer to my in-line comments. Thanks!

**Comment by nobbynguyen** [comment]
Sorry for being so clumsy.

**Pull request merged**